USING.md

```diff
@@ -25,14 +24,21 @@ The following modules are applications and data for testing and working with JTS
 To include JTS in a Maven project, add a dependency block like the following:
 
 ```xml
+<properties>
+    <jts.version>1.16.1</jts.version>
+</properties>
 <dependency>
     <groupId>org.locationtech.jts</groupId>
     <artifactId>jts-core</artifactId>
     <version>${jts.version}</version>
 </dependency>
 ```
 
-JTS snapshot artifacts are published to the LocationTech Maven repository. To include JTS in a project, add the following repositories to the pom.
+JTS artifacts are available on maven central.
+
+### Using JTS Snapshots
```

Comment:
Would it be good to refer to this link somewhere, since it's where the snapshots are found?

https://repo.locationtech.org/content/repositories/jts-snapshots/org/locationtech/jts/

Reply:
Specifying repo.locationtech.org should be enough.  I think the LT repo unifies all the various sub-repos into a consistent view, etc.